server/devdocs/__integration__/index.js

```diff
@@ -0,0 +1,146 @@
+jest.mock( 'config', () => ( {
```

Comment:
I renamed this file, but also updated it earlier. GitHub doesn't like it. It's easier to verify by commit. 

It's now moved to the integration tests.